src/app/client/src/app/app.component.ts

```diff
@@ -459,11 +463,11 @@ export class AppComponent implements OnInit, OnDestroy {
       this.consentConfig = { tncLink: '', tncText: this.resourceService.frmelmnts.lbl.nonCustodianTC };
       this.showGlobalConsentPopUpSection = true;
     }
-    this.checkFrameworkSelected();
   }
 
   public closeConsentPopUp() {
     this.showGlobalConsentPopUpSection = false;
+    this.checkFrameworkSelected();
```

Comment:
If you change this what will happen to the custodian user as this popup will not work

Reply:
that is handled in else case of
`public checkTncAndFrameWorkSelected() {
      if()else {
       this.checkFrameworkSelected();
     }
  }`